consensus/src/chained_bft/block_storage/block_store.rs

```diff
@@ -9,12 +9,13 @@ use crate::{
     counters,
     state_replication::StateComputer,
 };
+use anyhow::{bail, ensure, format_err, Context};
 use consensus_types::{
     block::Block, common::Payload, executed_block::ExecutedBlock, quorum_cert::QuorumCert,
     timeout_certificate::TimeoutCertificate, vote::Vote,
 };
+use debug_interface::event;
```

Comment:
This is unexpected? I see it in line 187/188 why did it work before but not now?

Reply:
No its not, if you look at the commit that introduces this, I remove the `#[macro_use] extern crate debug_interface` lines from the lib.rs file because we should try to avoid bringing all macros into scope where we can.